### ISSUE TITLE
Error: running README.md example code

### ISSUE THREAD

**Issue by ramfox** [rkyv/rkyv#258]
hey hey! Would love some guidance on how to get the example code to run.

Tried to run the `README.md` example code and got this error:
```sh
   Compiling rkyv_test v0.1.0 (.../rkyv_test)
error[E0425]: cannot find function `check_archived_root` in crate `rkyv`
   --> src/main.rs:35:26
    |
35  |     let archived = rkyv::check_archived_root::<Test>(&bytes[..]).unwrap();
    |                          ^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `archived_root`
    |
   ::: /Users/ramfox/.cargo/registry/src/github.com-1ecc6299db9ec823/rkyv-0.7.35/src/util/mod.rs:147:1
    |
147 | pub unsafe fn archived_root<T: Archive + ?Sized>(bytes: &[u8]) -> &T::Archived {
    | ------------------------------------------------------------------------------ similarly named function `archived_root` defined here
```

exact code run (essentially copy-paste from the readme, with `fn main` added):
`src/main.rs`
```rust
use rkyv::{Archive, Deserialize, Serialize};
// bytecheck can be used to validate your data if you want
use bytecheck::CheckBytes;

#[derive(Archive, Deserialize, Serialize, Debug, PartialEq)]
// This will generate a PartialEq impl between our unarchived and archived types
#[archive(compare(PartialEq))]
// To use the safe API, you have to derive CheckBytes for the archived type
#[archive_attr(derive(CheckBytes, Debug))]
struct Test {
    int: u8,
    string: String,
    option: Option<Vec<i32>>,
}

fn main() {
    let value = Test {
        int: 42,
        string: "hello world".to_string(),
        option: Some(vec![1, 2, 3, 4]),
    };

    // Serializing is as easy as a single function call
    let bytes = rkyv::to_bytes::<_, 256>(&value).unwrap();

    // Or you can customize your serialization for better performance
    // and compatibility with #![no_std] environments
    use rkyv::ser::{serializers::AllocSerializer, Serializer};

    let mut serializer = AllocSerializer::<0>::default();
    serializer.serialize_value(&value).unwrap();
    let bytes = serializer.into_serializer().into_inner();

    // You can use the safe API for fast zero-copy deserialization
    let archived = rkyv::check_archived_root::<Test>(&bytes[..]).unwrap();
    assert_eq!(archived, &value);

    // Or you can use the unsafe API for maximum performance
    let archived = unsafe { rkyv::archived_root::<Test>(&bytes[..]) };
    assert_eq!(archived, &value);

    // And you can always deserialize back to the original type
    let deserialized: Test = archived.deserialize(&mut rkyv::Infallible).unwrap();
    assert_eq!(deserialized, value);
}
```

**Comment by djkoloski** [comment]
It looks like you just need to enable the `validation` feature:

```toml
rkyv = { version = "0.7", features = ["validation"] }
```

I'll make a note of that in the README

**Comment by ramfox** [comment]
you rock, thank you!

**Comment by djkoloski** [comment]
Fixed by e8885e92b158a0ef7a5d7dac67872008fd580f90
